### ISSUE TITLE
微调的时候这个Train loss 怎么理解？

### ISSUE THREAD

**Issue by lzpsqzr** [linyiLYi/bilibot#27]
![image](https://github.com/linyiLYi/bilibot/assets/12146453/e9295323-7cc0-4d2f-8362-9187a0746801)
微调的时候这个Train loss 怎么理解？然后上面的val loss又是什么意思？

**Comment by theblackcat102** [comment]
确保 train loss 持续下降即可，val loss 就是验证资料集的表现，如果后续一直都在上升那就调低 learning rate试试看

**Comment by brucewayne798** [comment]
损失，就是预测内容和真实内容的差距
